workshops/views.py

```diff
@@ -474,6 +497,13 @@ def task_details(request, task_id):
     return render(request, 'workshops/task.html', context)
 
 
+@login_required
+def task_delete(request, task_id):
```

Comment:
A usual CRUD would redirect to the list of objects of the same kind after deleting one object from that list.

> Docstring's should explain what a function (or whatever) does, not who's using it.

In case of deleting a task, that function redirects to the task's event. The redirect is there to help with inline edits from the event's page. I think this is unusual enough to at least deserve a comment. This redirect is also a part of function's behavior, so it may just as well be put in the docstring.

> It's better to pick distinct names

Personally I'd use a different name and URL for that view - or jump to "RESTful" fashion and return a different status code instead of the redirect.